doc/modules/changes/20170609_gassmoeller

```diff
@@ -0,0 +1,5 @@
+New: A new initial temperature condition was added that adds a random
+noise of user prescribed magnitude to the initial temperature field.
```

Comment:
a random noise -> random noise

(noise is like milk -- it has no singular or plural)